tracing-core/src/test/groovy/io/micronaut/tracing/instrument/util/MDCReactorSpec.groovy

```diff
@@ -53,6 +54,7 @@ class MDCReactorSpec extends Specification {
     @AutoCleanup
     HttpClient client = HttpClient.create(embeddedServer.URL)
 
+    @Ignore("https://github.com/micronaut-projects/micronaut-core/issues/8617")
```

Comment:
@n0tl3ss  can you change this to` @PendingFeature` instead? That way, when #8167 is fixed the test will start passing and it will break the build and we will remember to remove `@PendingFeature`